.travis.yml

```diff
@@ -88,6 +88,7 @@ install:
   - git clone https://github.com/openresty/lua-resty-mysql.git ../lua-resty-mysql
   - git clone https://github.com/openresty/stream-lua-nginx-module.git ../stream-lua-nginx-module
   - git clone -b v2.1-agentzh https://github.com/openresty/luajit2.git luajit2
+  - git clone https://github.com/openresty/lua-resty-signal.git ../lua-resty-signal
```

Comment:
I'm not feeling comfortable adding this new dependency for the test suite. Can we avoid it? Maybe you can just use `os.execute("kill -HUP " .. pid)` or something like that?

Reply:
OK